mantidproject/mantid
# Created Initial Calibration Tab for New Engineering Diffractio…

**Description of work.**

Added minimal Calibration tab for new python based Engineering Diffration interface

**To test:**
- Check the GUI opens successfully in plot and workbench.
- Check calibration works without errors (Van: 307521 Cal: 305738 Are valid run numbers)
- Check 4 plots are created when check box is ticked. 
- Check browse buttons work as expected.
- Check providing an RB number creates a User directory. 
- Check instrument can be changed without errors and the run number entries search with the right file prefix. 
- Check tabbing order makes sense.

Closes #26846

*This does not require release notes* because **it is not yet in a state where it would be useful to users.**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
